MomsSpaghetti/Pages/Index.cshtml

```diff
@@ -83,194 +82,182 @@
-                                        <img src="img/test-1.jpg" alt="Test">
-                                    </figure>
-                                    <div class="likes__data">
-                                        <h4 class="likes__name">Pasta with Tomato ...</h4>
-                                        <p class="likes__author">The Pioneer Woman</p>
-                                    </div>
-                                </a>
-                            </li>
-                            -->
-                        </ul>
-                    </div>
-                </div>
-            </header>
 
 
-            <div class="results">
-                <ul class="results__list">
 
-                    @{if (Model.SearchRecipes.Result != null)
-                        {
 
+                    <div class="results">
+                        <ul class="results__list">
 
-                            @*
-
-                                    var props = Model.SearchRecipes.Result.Properties();
-                                    var propsArray = props.ToArray();
-                                    var first = propsArray[1].Value[0]["title"];
-
-                                <h1>@Model.SearchRecipes.Result["recipes"][0]["title"]</h1>
-                                <h1>@Model.SearchRecipes.Result["recipes"][1]["title"]</h1>
-                            *@
-
-                            foreach (var recipe in Model.SearchRecipes.Result["recipes"])
-                            {
-                                <li>
-                                    <a class="results__link results__link--active" href="?recipeId=@recipe["recipe_id"]">
-                                        <figure class="results__fig">
-                                            <img src="@recipe["image_url"]" alt="Test">
-                                        </figure>
-                                        <div class="results__data">
-                                            <h4 class="results__name">@recipe["title"]</h4>
-                                            <p class="results__author">@recipe["author"]</p>
-                                        </div>
-                                    </a>
-                                </li>
-                            }
+                            @{if (Model.SearchRecipes.Result != null)
+                                {
 
 
-                        }
+                                    @*
 
+                            var props = Model.SearchRecipes.Result.Properties();
+                            var propsArray = props.ToArray();
+                            var first = propsArray[1].Value[0]["title"];
 
-                    }
+                        <h1>@Model.SearchRecipes.Result["recipes"][0]["title"]</h1>
+                        <h1>@Model.SearchRecipes.Result["recipes"][1]["title"]</h1>
+                                    *@
```

Comment:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.